dotcom-rendering/src/web/components/CommercialMetrics.importable.tsx

```diff
@@ -39,23 +38,26 @@ const CommercialMetricsWithAB = ({ enabled }: { enabled: boolean }) => {
 		const shouldForceMetrics = ABTestAPI.allRunnableTests(tests).some(
 			(test) => testsToForceMetrics.map((t) => t.id).includes(test.id),
 		);
-		const userIsInSamplingGroup = Math.random() <= 1 / 100;
+
 		const isDev =
 			window.guardian.config.page.isDev ||
 			window.location.hostname.includes('localhost');
 
-		if (isDev || shouldForceMetrics || userIsInSamplingGroup) {
-			sendCommercialMetrics(
-				pageViewId,
-				browserId || undefined,
-				isDev,
-				adBlockerInUse,
-			);
-			// TODO: capture CWV also, to ensure commercial performance
-			// doesn’t come at the expense of user experience.
-			// See https://git.io/JP68Q in `frontend`
+		initCommercialMetrics(
+			pageViewId,
+			browserId || undefined,
```

Comment:
I'm sure it is possible. In the interest of learning, what is the reason you'd prefer it inside the `useOnce`?

Reply:
Oh just narrowing scope and moving 'work' closer to where/when its required rather than up front.

It doesn't look like those values are used anywhere else in this component.

Reply:
Actually, an update to this is that pageViewId needs to be outside the useOnce because it's part of the state that the hook is keeping track of (in the array that's the second argument of the hook. So I'm wondering whether it's better to keep it all together for clarity.

Reply:
Ah okay. I think however the only reason it is a dependency is because it's outside the `useOnce` and I don't think it needs to be and could be derived inside.

Anyway just a minor tangent - it could be a follow up, totally fair if you want to keep the PR focussed 👍

Reply:
I think moving these variables closer to where they are being used is fair. If they're not used elsewhere then it's always nice to keep things together like this.

In terms of changeability, these two values are static and won't change for the life of a page load